.test-infra/metrics/build_and_publish_containers.sh

```diff
@@ -36,8 +33,8 @@ fi
 
 echo
 echo ===========Start==========
-CONTAINER_VERSION_NAME=$1
-DO_PUSH=$2
+DO_PUSH=$1
+CONTAINER_VERSION_NAME=${2:-latest}
```

Comment:
will latest overwrite container image version? If not, we should add cleanup script for old versions.

Reply:
It won't.
We have a script that removes all images older than 24 hours from Jenkins workers: https://github.com/apache/beam/blob/a13ef2491952d51bbab40c0fea3974c46f43edd2/.test-infra/jenkins/job_Inventory.groovy#L69

But, as far as I know, we don't have a similar cleanup task for images that were pushed to Container Registry (gcr)

Reply:
One possible solution would be to create a cron job that would remove untagged images on Container Registry (very similar to this one: https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_CancelStaleDataflowJobs.groovy). WDYT?

Reply:
Could be done in a separate PR.

Reply:
@Ardagan

Reply:
Since we already have a script, it should be fine.